solutions/devicesimulation-nohub/single-vm/docker-compose.yml

```diff
@@ -60,3 +63,13 @@ services:
       - PCS_AUTH_AUDIENCE
       - PCS_AUTH_REQUIRED
       - PCS_CORS_WHITELIST
+
+  diagnostics-svc:
+    image: azureiotpcs/pcs-diagnostics-dotnet:millennium
+    environment:
+      - PCS_LOG_LEVEL
+      - PCS_AUTH_ISSUER
+      - PCS_AUTH_AUDIENCE
+      - PCS_AUTH_REQUIRED
+      - PCS_CORS_WHITELIST
+      - PCS_SOLUTION_ID
```

Comment:
shouldn't this be "deployment id" ?

Reply:
I called it solution Id to keep it consistent with solution name and solution type.